src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -851,7 +851,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 					if (label.length === 0 || commandMap.has(label)) {
 						continue;
 					}
-					let description = `${entry.cwd}`;
+					let description = collapseTildePath(entry.cwd, this._userHome, this._processManager?.os === OperatingSystem.Windows ? '\\' : '/');
```

Comment:
will this do the right thing for WSL?

Reply:
No it won't, but it won't do the right thing for any regular ssh with shell integration either. https://github.com/microsoft/vscode/issues/154583